RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+View.swift

```diff
@@ -24,9 +24,9 @@ extension DisplayEntitiesControlledByMnemonic {
 		public var body: some SwiftUI.View {
 			WithViewStore(store, observe: { $0 }, send: { .view($0) }) { viewStore in
 				VStack(alignment: .leading) {
-					if viewStore.displayRevealMnemonicLink {
+					if viewStore.mode == .mnemonicCanBeDisplayed {
```

Comment:
Why not just switch?

Reply:
I dont like having to use `EmptyView`. but sure can use that - there is not view for `displayAccountListOnly`

Reply:
happy now? https://github.com/radixdlt/babylon-wallet-ios/pull/860/commits/89b223e43d17f7531112b251b8d334e3f6a19444 :D